staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -224,9 +224,9 @@ type SecureServingInfo struct {
 	// Values are from tls package constants (https://golang.org/pkg/crypto/tls/#pkg-constants).
 	MinTLSVersion uint16
 
-	// CipherSuites optionally overrides the list of allowed cipher suites for the server.
+	// TLSCipherSuites optionally overrides the list of allowed cipher suites for the server.
 	// Values are from tls package constants (https://golang.org/pkg/crypto/tls/#pkg-constants).
-	CipherSuites []uint16
+	TLSCipherSuites []uint16
```

Comment:
missed that this renamed... since this is already in the SecureServingInfo struct, no need to prefix with TLS. revert this rename